base_name_search_improved/views/ir_model.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Daniel Reis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
```

Comment:
s/`<openerp>`/`<odoo>`/ and remove the `<data>` tag (but it's not a blocking point)